anaDACScan.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+"""
+anaDACScan
+==============
+"""
+import ROOT as r
+
+from gempython.gemplotting.utils.anautilities import parseCalFile
+from gempython.utils.nesteddict import nesteddict
+from gempython.gemplotting.utils.anautilities import make3x8Canvas
+
+import argparse
```

Comment:
You have no other functions declared in this script, put these imports inside `main`

Reply:
done